lib/trento_web/views/v1/activity_log_view.ex

```diff
@@ -15,4 +18,29 @@ defmodule TrentoWeb.V1.ActivityLogView do
       occurred_on: entry.inserted_at
     }
   end
+
+  def render("pagination.json", %{pagination: pagination}) when is_map(pagination) do
```

Comment:
If the guard is not strictly necessary, let's remove it.
```suggestion
  def render("pagination.json", %{pagination: pagination}) do
```